sharepoint/sharepoint-ps/sharepoint-online/Get-SPOContainerType.md

```diff
@@ -0,0 +1,81 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spocontainertype
+applicable: SharePoint Online
+title: Get-SPOContainerType
+schema: 2.0.0
+author: ShreyasSar26
+ms.author: ShreyasSar26
+ms.reviewer:
+---
+
+# Get-SPOContainerType
+
+## SYNOPSIS
+
+Returns one or more container types created in the tenant. 
+
+## SYNTAX
+
+### ParamSet1
+
+```powershell
+Get-SPOContainerType 
+```
+### ParamSet2
+```powershell
+Get-SPOContainerType [-ContainerTypeId <ContainerTypeId>] 
+```
+
+## DESCRIPTION
+
+This cmdlet returns all the container types present in the tenant or details of a specific container type when paired with the containertype ID parameter. 
+
+You must be a SharePoint Embedded administrator to run the cmdlet.
+
+For permissions and the most current information about Windows PowerShell for SharePoint Embedded Containers, see the online documentation at [Intro to SharePoint Embedded Containers Management Shell](/powershell/sharepoint/sharepoint-online/introduction-sharepoint-online-management-shell).
+
+## EXAMPLES
+
+### Example 1
+
+```powershell
+Get-SPOContainerType 
+```
+
+Example 1 retrieves all the container types present in the tenant and displays information such as Container Type ID, Container Type Name, Owning Application ID, billing details, creation date, expiry date.
```

Comment:
> and displays information such as Container Type ID, Container Type Name, Owning Application ID, billing details, creation date, expiry date [](http://example.com/codeflow?start=65&length=140)

remove this entire section -- no need to call out the properties returned as users will see when they execute the cmdlet. Also, these properties could change in the future -- means you'd have to keep updating this documentation.

Reply:
Closed